src/data/builder-tools.js

```diff
@@ -1138,6 +1138,22 @@ export const Showcases = [
     website: "https://github.com/SAIB-Inc/Cardano.Sync",
     getstarted: null,
     tags: ["net", "sdk", "chainindex"]
+  },
+  {
+    title: "Cardano Looking Glass",
+    description: "A visual blockchain explorer which supports CIP54 and CIP68 Smart NFTs.",
+    preview: require("./builder-tools/clg.png"),
+    website: "https://clg.wtf/",
```

Comment:
Site not available (NSURLErrorCannotFindHost)

Reply:
I'm not sure if this was a transient error, but the site is definitely up, and indexed by Google which is reporting no problems with access. I've just checked it with this random tool and it verified that the site is up:
https://www.isitdownrightnow.com/clg.wtf.html

Are you still unable to access this URL?

Reply:
@katomm from the last reported downtime it looks reliable enough & that the problem was a glitch rather than an overall service problem.